src/ledgerhelpers/programs/addtrans.py

```diff
@@ -146,7 +145,6 @@ def accounts_and_last_commodities_loaded(self, accounts, last_commos):
             self.status.set_text("")
 
     def journal_load_failed(self, e):
-        traceback.print_exc()
```

Comment:
This used to work.  What if we replace this with the function from the `traceback` module that uses the last traceback?  https://docs.python.org/3/library/traceback.html#traceback.print_last

Reply:
Note that `print_last` is documented to only work if the exception hits the interpreter uncaught.

As expected, changing to that just causes:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/ledgerhelpers/programs/addtrans.py", line 149, in journal_load_failed
    traceback.print_last()
  File "/usr/lib/python3.10/traceback.py", line 189, in print_last
    raise ValueError("no last exception")
ValueError: no last exception
```

Reply:
Aaaaaa.  Is there any way to retrieve the traceback from `e`?

Reply:
Oh yes there is!  This works on Python 3.10.

https://docs.python.org/3/library/traceback.html#traceback.print_exception

Please use this instead.  It may not be optimal in Python 3.9 and lower, but going forward it is the optimal way to print the correct exception traceback.

Reply:
Done.